makefiles/periph.mk

```diff
@@ -0,0 +1,15 @@
+MODULE ?= periph
+
+# enable submodules
+SUBMODULES := 1
+
+# don't fail if requested submodule does not have corresponding *.c file
+SUBMODULES_NOFORCE := 1
+
+# in case of e.g., <cpu>_periph_common, use periph_* as submodule prefix
+BASE_MODULE := periph
+
+# periph_common matches the submodule selection, but shouldn't. So filter out here.
+USEMODULE := $(filter-out periph_common, $(USEMODULE))
```

Comment:
is this really needed? As I understood the `SUBMODULES_NOFORCE` option, Makefile.base skips entries where it can't find c-files for anyway. So it should not make a difference if `periph_common` is in that list as there is most likely no c-file with that name available anyway, right?